lib/util.js

```diff
@@ -242,6 +243,10 @@ inspect.styles = Object.assign(Object.create(null), {
   // "name": intentionally not styling
   'regexp': 'red'
 });
+// Magenta has problems with the deafult configuration of Powershell
+// Ref: https://github.com/nodejs/node/issues/14243
+if (process.platform === 'win32')
+  inspect.styles.date = 'magenta;bold';
```

Comment:
OTOH, such an rewrite of colors can be considered breaking. Probably better to not go that route and instead make sure we don't print that magenta, and maybe that modules like chalk workaround it until PS fixes it.